File: 2025/intermediario/2025.1/1040.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){
    double nota1, nota2, nota3, nota4, notaExame, media;
    cin >> nota1 >> nota2 >> nota3 >> nota4;
    media = ((nota1 * 2) + (nota2 * 3) + (nota3 * 4) + (nota4 * 1)) / (2 + 3 + 4 + 1);
    cout << fixed << setprecision(1);
    cout << "Media: "<< media << "\n";
    if (media >= 7.0){
        cout << "Aluno aprovado." << "\n";
    } else if (media >= 5.0 && media <= 6.9){
        cout << "Aluno em exame." << "\n";
        cin >> notaExame;
        cout << "Nota do exame: " << notaExame << "\n";
        media = (media + notaExame) / 2;
        if (notaExame >= 5.0){
            cout << "Aluno aprovado." << "\n";
            cout << "Media final: " << media << "\n";
        } else {
            cout << "Aluno reprovado." << "\n";
            cout << "Media final: " << media << "\n";
        }
    } else {
        cout << "Aluno reprovado." << "\n";
    }

    return 0;
}

File: 2025/iniciante/2025.1/14-04-2025/1040.py
N1, N2, N3, N4 = map(float, input().split())
media = ((N1 * 2) + (N2 * 3) + (N3 * 4) + (N4 * 1)) / 10
if media >= 7.0:
    print(f'Media: {media:.1f}')
    print('Aluno aprovado.')
elif media < 5.0:
    print(f'Media: {media:.1f}')
    print('Aluno reprovado.')
else:
    print(f'Media: {media:.1f}')
    print('Aluno em exame.')
    notaexame = float(input())
    print(f'Nota do exame: {notaexame:.1f}')
    mediafinal = (notaexame + media) / 2
    if mediafinal >= 5.0:
        print('Aluno aprovado.')
        print(f'Media final: {mediafinal:.1f}')
    else:
        print('Aluno reprovado.')
        print(f'Media final: {mediafinal:.1f}')